i18n/jpn/src/asciidoctorWebViewConverter.i18n.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@YoshihideShirai I'm using English as placeholder, feel free to update these files